src/Admin/AdminInterface.php

```diff
@@ -564,130 +447,106 @@ public function getSubClasses(): array;
      *
      * @param $subClass
      */
-    public function addSubClass($subClass);
+    public function addSubClass($subClass): void;
 
     /**
      * Sets the list of supported sub classes.
      */
-    public function setSubClasses(array $subClasses);
+    public function setSubClasses(array $subClasses): void;
 
     /**
      * Returns true if the admin has the sub classes.
      *
      * @param string $name The name of the sub class
-     *
-     * @return bool
      */
-    public function hasSubClass($name);
+    public function hasSubClass(string $name): bool;
 
     /**
      * Returns true if a subclass is currently active.
-     *
-     * @return bool
      */
-    public function hasActiveSubClass();
+    public function hasActiveSubClass(): bool;
 
     /**
      * Returns the currently active sub class.
      *
      * @return string the active sub class
      */
-    public function getActiveSubClass();
+    public function getActiveSubClass(): string;
 
     /**
      * Returns the currently active sub class code.
      *
      * @return string the code for active sub class
      */
-    public function getActiveSubclassCode();
+    public function getActiveSubclassCode(): string;
 
     /**
      * Returns the list of batchs actions.
      *
      * @return array<string, mixed> the list of batchs actions
      */
-    public function getBatchActions();
+    public function getBatchActions(): array;
 
     /**
      * Returns Admin`s label.
-     *
-     * @return string
      */
-    public function getLabel();
+    public function getLabel(): ?string;
 
     /**
      * Returns an array of persistent parameters.
      *
      * @return array<string, mixed>
      */
-    public function getPersistentParameters();
+    public function getPersistentParameters(): array;
 
     public function getPersistentParameter(string $name);
 
     /**
      * Set the current child status.
-     *
-     * @param bool $currentChild
      */
-    public function setCurrentChild($currentChild);
+    public function setCurrentChild(bool $currentChild): void;
 
     /**
      * Returns the current child status.
      *
      * NEXT_MAJOR: Rename the function isCurrentChild()
-     *
-     * @return bool
      */
-    public function getCurrentChild();
+    public function getCurrentChild(): bool;
 
     /**
      * Get translation label using the current TranslationStrategy.
-     *
-     * @param string $label
-     * @param string $context
-     * @param string $type
-     *
-     * @return string
      */
-    public function getTranslationLabel($label, $context = '', $type = '');
+    public function getTranslationLabel(string $label, string $context = '', string $type = ''): string;
 
     /**
      * @param object $object
-     *
-     * @return MetadataInterface
      */
-    public function getObjectMetadata($object);
+    public function getObjectMetadata($object): MetadataInterface;
```

Comment:
```suggestion
    public function getObjectMetadata(object $object): MetadataInterface;
```

Reply:
This argument is passed to `toString()`, which seems to accept `null`.